### ISSUE TITLE
Does Tantivy also specialize SIMD for aarch64?

### ISSUE THREAD

**Issue by mikemccand** [Tony-X/search-benchmark-game#17]
Arm (aarch64) cpus are becoming more popular/powerful (e.g. AWS Graviton instances) ... does Tantivy also specialize SIMD for Arm CPUs?

**Comment by mikemccand** [comment]
We could also simply run the current benchmarks on a Graviton2 instance?

**Comment by fulmicoton** [comment]
We do not use much handrolled simd stuff these days. I assume you mean for decompression? Some contributors added a NEON implementation for bitpacking, although it is not that beneficial.

I'd love to see have graviton figures

**Comment by Tony-X** [comment]
> does Tantivy also specialize SIMD for Arm CPUs?

I don't think so. However, I think we can have *some* hope for LLVM to figure out using SIMD wherever it finds suitable

**Comment by fulmicoton** [comment]
@Tony-X see my reply above.
https://github.com/quickwit-oss/bitpacking/blob/master/src/bitpacker4x.rs#L71-L120

**Comment by Tony-X** [comment]
Ah! I missed it. @fulmicoton thanks for the pointer
